docs/src/main/paradox/server-class.md

```diff
@@ -0,0 +1,117 @@
+The main class
+----------------
+
+Let's dissect the main class, `QuickstartServer`. We make this class runnable by extending `App` (we will discuss the trait `JsonSupport` later):
+
+@@snip [QuickstartServer.scala]($g8src$/scala/com/lightbend/akka/http/sample/QuickstartServer.scala) { #main-class }
+
+Now that we have a class to run we should add some Akka HTTP fundamentals with which we will build our RESTful web service:
+
+* define routes bound to endpoints and HTTP directives
+* create a server bound to an IP and port that will handle all requests
+* add error handling for when something goes wrong
+
+Let us take a look at each of these steps here below.
+
+## Routes
+
+For our service we want to define the following endpoints:
+
+| Path        | Http directive  | Intent             | Returns              |
+|-------------|-----------------|--------------------|----------------------|
+| /user       | POST            | Create a new user  | Confirmation message |
+| /user/$ID   | GET             | Retrieve a user    | JSON payload         |
+| /user/$ID   | DELETE          | Remove a user      | Confirmation message |
+| /users      | GET             | Retrieve all users | JSON payload         |
+
+Akka HTTP provides a [domain-specific language](https://en.wikipedia.org/wiki/Domain-specific_language) (DSL) to simplify the routes/endpoints definition. Each route is composed of one or more `akka.http.scaladsl.server.Directives`, e.g. `path`, `get`, `post`, `complete`, etc.
```

Comment:
add below here: "If you would rather use the raw underlying APIs, that is also possible and refered to as [low-level API](http://doc.akka.io/docs/akka-http/current/scala/http/low-level-server-side-api.html)"?